neqo-transport/src/connection/mod.rs

```diff
@@ -1101,10 +1101,18 @@ impl Connection {
     #[must_use = "Output of the process function must be handled"]
     pub fn process(&mut self, dgram: Option<&Datagram>, now: Instant) -> Output {
         if let Some(d) = dgram {
+            #[cfg(feature = "build-fuzzing-corpus")]
+            neqo_common::write_item_to_fuzzing_corpus("packet", d);
             self.input(d, now, now);
             self.process_saved(now);
         }
-        self.process_output(now)
+        #[allow(clippy::let_and_return)]
+        let output = self.process_output(now);
+        #[cfg(feature = "build-fuzzing-corpus")]
+        if let Some(d) = output.clone().dgram() {
+            neqo_common::write_item_to_fuzzing_corpus("packet", &d);
```

Comment:
I would also check here whether the test frame writer was enabled and not use packets where it was on.

I know that we might want to have a corpus that also includes invalid stuff, but it probably won't help to give the fuzzer invalid garbage.